src/main/java/com/moabam/api/domain/entity/Member.java

```diff
@@ -72,7 +72,7 @@ public class Member extends BaseTimeEntity {
 
 	@Enumerated(EnumType.STRING)
 	@Column(name = "role", nullable = false)
-	@ColumnDefault("USER")
+	@ColumnDefault("`USER`")
```

Comment:
이거 백틱이 아니라 작은 따옴표 써야하지 않나여?
```suggestion
	@ColumnDefault("'USER'")
```